@here/harp-mapview/lib/Utils.ts

```diff
@@ -377,24 +404,90 @@ export namespace MapViewUtils {
             distance: unconstrained.distance
         };
 
+        if (projection.type === ProjectionType.Planar) {
+            if (scale > 1) {
```

Comment:
I know it's not your code but can you please add a comment why we have to distinguish between `scale <= 1` and `scale > 1`

Reply:
it's actually my code, merged a couple of weeks ago. I'll add the comment.